datacenter/dtr/2.4/guides/admin/monitor-and-troubleshoot/notary-audit-logs.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Check Notary audit logs
+description: When you push signed images, Docker Trusted Registry keeps audit
+  logs for the changes made to the image metadata. Learn how to view these logs.
+keywords: registry, monitor, troubleshoot
+---
+
+DTR keeps audit logs of changes made to trusted repositories.
+Every time you push a signed image to a repository, or delete trust data for a
+repository, DTR logs that information.
+
+These logs are only available from the DTR API.
+
+## Get an authentication token
+
+To access the audit logs you need to authenticate your requests using an
+authentication token. You can get an authentication token for all repositories,
+or one that is specific to a single repository.
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#tab1">Global</a></li>
+  <li><a data-toggle="tab" data-target="#tab2">Repository-specific</a></li>
+</ul>
+<div class="tab-content">
+  <div id="tab1" class="tab-pane fade in active" markdown="1">
+
+```bash
+curl --insecure --silent \
+--user <user>:<password> \
+"https://<dtr-url>/auth/token?realm=dtr&service=dtr&scope=registry:catalog:*"
+```
+
+  <hr></div>
+  <div id="tab2" class="tab-pane fade" markdown="1">
+
+```bash
+curl --insecure --silent \
+--user <user>:<password> \
+"https://<dtr-url>/auth/token?realm=dtr&service=dtr&scope=repository:<dtr-url>/<repository>:pull"
+```
+
+  <hr></div>
+</div>
+
+DTR returns a JSON file with a token, even when the user doesn't have access
```

Comment:
We should probably call out that the token still doesn't grant access to the repository.